internal/provider/cloudspace_resource.go

```diff
@@ -63,6 +63,23 @@ func (r *cloudspaceResource) Configure(ctx context.Context, req resource.Configu
 }
 
 func (r *cloudspaceResource) ModifyPlan(ctx context.Context, req resource.ModifyPlanRequest, resp *resource.ModifyPlanResponse) {
+	if req.Plan.Raw.IsNull() {
+		// resource is being destroyed
+		return
+	}
+	if req.State.Raw.IsNull() && !req.Plan.Raw.IsNull() {
```

Comment:
> !req.Plan.Raw.IsNull()

This is not needed here right? We've already returned early above if `Plan.Raw` is null? So here its always not null?

Reply:
This change has been reverted.